overlord/configstate/configcore/homedirs_test.go

```diff
@@ -168,12 +168,16 @@ func (s *homedirsSuite) TestConfigureUnchangedConfig(c *C) {
 }
 
 func (s *homedirsSuite) TestConfigureApparmorTunableFailure(c *C) {
+	restore := apparmor.MockLevel(apparmor.Full)
```

Comment:
I wonder if we should do the mocking on a suite level instead and just switch it to apparmor.NONE (or whatever it was) in the test that checks that the non-apparmor handling is correct?

Reply:
Good suggestion, fixed!